usr/sbin/p11sak/p11sak.c

```diff
@@ -4533,7 +4566,25 @@ static CK_BBOOL objclass_expected(CK_OBJECT_HANDLE obj, enum p11sak_objclass obj
 
     return CK_FALSE;
 }
-
+#if defined(_AIX)
```

Comment:
So does this function only differ in 2 lines? I.e. no `void *private` of `compare_obj`, and `void *priv` instead of `void *private` ? Maybe it is better readable if you just ifdef those single lines? 

Also, if you name the global `private` variable as `global_private` then you wont need to name the function parameter differently, and its easier to understand when you access the global variable versus the local one.